lib/checkclass.cpp

```diff
@@ -2402,24 +2402,24 @@ void CheckClass::checkCopyCtorAndEqOperator()
         if (!hasNonStaticVars)
             continue;
 
-        int hasCopyCtor = 0;
-        int hasAssignmentOperator = 0;
+        int copyCtors = 0;
```

Comment:
it would make more sense that these were enums like:

    NO, WITHOUT_BODY, WITH_BODY

Reply:
Ok.